### ISSUE TITLE
Crying Obsidian

### ISSUE THREAD

**Issue by unown006** [EarthMC/Issue-Tracker#1668]
### Is your feature request related to a problem? Please describe.

Currently It is almost impossible to obtain crying obsidian. The only way you can is bartering with pilins exchanging gold ingots for a 8.7% chance (according to the wiki) to obtain 1-3. This item is incredibly expensive because the EMC economy is tied to Gold.

### Describe the solution you'd like

I would like to create a custom recipe for crying obsidian. 

9 obsidian = 1 crying obsidian

**Comment by FoxStarWar** [comment]
I think this is a great idea +1

**Comment by OxyIncluded** [comment]
I also Thing this Needs to be added I have builds that require alot of this stuff

**Comment by Thendzombie** [comment]
Amazing Idea why dont we already have this?

**Comment by Toadness** [comment]
I like this idea the crafting cost seems a little high but other then that seems solid

**Comment by SamuelBladeburn** [comment]
Please add this ty!!!

**Comment by PankakeBeeb** [comment]
Would be really nice if added

**Comment by Fruitloopins** [comment]
top 10 mass alters in earthmc history

**Comment by unown006** [comment]
> And not one of those Alts (allegedly...) suggested a more Minecraft-y feeling recipe with a better cost...
> 
> Like 1 Obsidian surrounded by 8 Ghast Tears equals either 1 Crying Obsidian (if you wanted to keep the price of CO at the current 3g) or 2 or 3 CO (if you want to drop the price to 2g or 1g per CO).

The main purpose is to not have it tied to gold the only reason the current cost is as high as it is, IS because the current recipe is tied to gold. So changing the Recipe will have the effect that the cost of CO will go Down on the current market because the market is inflated and we shouldn't balance the recipe around a already inflated market.

I hope that makes sense if not I can try and explain it further.

**Comment by jagster104** [comment]
dew it pls

**Comment by 0rremorre** [comment]
Do the thing

**Comment by Aylywyn** [comment]
It's not tied to gold...  it's only tied to gold if good is in  the recipe.

Right now, the supply is fixed and non-renewable.  It's 3g per block because of that.  Obsidian and Ghast Tears are both renewable so there is an infinite supply.

I mention 1Ob+8GT=1/2/3CO giving prices of 3g, 2g, or 1g because currently Obsidian is 64/g at worst and Ghast Tears are 3 or 4/g and that adds up to the price of about 3g.  If the recipe gives 1 you charge 3g, if 2 then 2g, if 3 then 1g per crying obsidian.

Your 9 Obsidian equals a Crying Obsidian would give a price of maybe 8CO/1g.  Much cheaper but the materials still set the price.

It's not intrinsically tied to gold, it has become renewable though, and the components cost gold, like anything else.

It would be similar if, say, 5 Horn Coral Blocks plus 4 Smooth Quartz equaled one Sponge...

**Comment by unown006** [comment]
> It's not tied to gold... it's only tied to gold if good is in the recipe.

**Which is currently the case by trading with Pilins**

> 
> Right now, the supply is fixed and non-renewable. It's 3g per block because of that. Obsidian and Ghast Tears are both renewable so there is an infinite supply.

**Yes**

> 
> I mention 1Ob+8GT=1/2/3CO giving prices of 3g, 2g, or 1g because currently Obsidian is 64/g at worst and Ghast Tears are 3 or 4/g and that adds up to the price of about 3g. If the recipe gives 1 you charge 3g, if 2 then 2g, if 3 then 1g per crying obsidian.

**This is where I think we Differ while your purposed recipe achieves making the resource not tied to gold you are balancing it with a inflated market. It seems like you think because CO is priced high because its almost impossible to obtain it should still be priced High with new inputs. I don't agree with this because the market itself is artifichly high (inflated) not because of market trends but because the resource is tied to gold.**

> 
> Your 9 Obsidian equals a Crying Obsidian would give a price of maybe 8CO/1g. Much cheaper but the materials still set the price.

**Yes**


> 
> It's not intrinsically tied to gold, it has become renewable though, and the components cost gold, like anything else.

**Your purposed recipe is not tied to gold but the current way to obtain CO is**

> 
> It would be similar if, say, 5 Horn Coral Blocks plus 4 Smooth Quartz equaled one Sponge...

**Comment by Aylywyn** [comment]
Oh, right Piglin Trading...  I forget that as I've never really used it, and with gold as currency...

CO is expensive but technically renewable over the supply that was in chest loot and a rare portal.  Every 11g you should get 2, on average plus the other stuff you get...  Yeah, 3g sounds right as is.

You can pick a recipe for almost any price but should also be a theme, more Minecraft-y.  And it should take the value, both in gold and usefulness, into account.

Obsidian plus Tears is Crying Obsidian.  Make it as low as 1 Obsidian plus 1 Tear equals 1 CO... That's about 3CO/1g.

You don't necessarily want to crash the existing Market though for economic reasons.  A way more dramatic example is for Sponges above, dropping them from 7g to... uh... 1g or so wipes out 30Kg.  But still an issue with Crying Obsidian.

**Comment by carrotcake2002** [comment]
good idea

**Comment by Proserrr** [comment]
Should be added along with purpur imo

**Comment by Aylywyn** [comment]
Yeah,  I could see no custom recipe if it was non-renewable like Sponge and you wanted an Item people could hoard and fight over. But Crying Obsidian (just pick a price...) and Purpur (Stone Brick + Lapis + RSDust?), are strangely left out of Customs.

**Comment by AGR280** [comment]
Maybe have the crafting recipe be 1 obsidian in the center, a ghast tear under, and two amethyst shards on both sides for one crying obsidian. It would make it resonably craftable but still somewhat hard to get due to you needing ghast tears.

**Comment by AGR280** [comment]
Maybe have the crafting recipe be 1 obsidian in the center, a ghast tear under, and two amethyst shards on both sides for one crying obsidian. It would make it resonably craftable but still somewhat hard to get due to you needing ghast tears.

**Comment by CorruptedGreed** [comment]
It will remain tied to Bartering for now.
